bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gaia-list",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
I think we should only stamp versions once PRs have landed on master. Patch version should be stamped using:

```
bower version patch
```